src/components/common/OfficeAddresses.vue

```diff
@@ -75,16 +77,16 @@
                   v-model="inheritDeliveryAddress"
                 />
               </div>
-              <div class="address-wrapper"
-                v-if="!isSame(deliveryAddress, mailingAddress, 'actions') || showAddressForm"
+              <div class="address-wrapper" v-if="showAddressForm ||
+                !isSame(deliveryAddress, mailingAddress, ['actions','addressType'])"
```

Comment:
All addresses now always have actions and addressType properties, so need to omit both in this comparison.

Reply:
NB: for like-to-like comparison we don't need to omit address type, but here it's comparing a delivery and mailing address.